src/main/java/com/f2prateek/bee/PhoneStateReceiver.kt

```diff
@@ -8,7 +8,9 @@ import android.content.Context.NOTIFICATION_SERVICE
 import android.content.Intent
 import android.support.v4.app.NotificationCompat
 import android.telephony.TelephonyManager
-import android.telephony.TelephonyManager.*
+import android.telephony.TelephonyManager.EXTRA_STATE_IDLE
```

Comment:
IIRC, the default Kotlin code style in IJ uses star imports. there's a box to uncheck somewhere in there.